packages/neuron-ui/src/utils/calculateFee.ts

```diff
@@ -1,5 +1,5 @@
 export const calculateFee = (tx: any) => {
-  if (!tx) {
+  if (!tx?.inputs || !tx?.outputs) {
```

Comment:
In what case `!tx` is not enough as a condition

Reply:
Reset. I mean `tx.inputs` maybe a little bit more precise.

Reply:
When we didn't give the correct `address` or `account`, we cannot earn the right data. That may cause some errors like this. 
<img width="1383" alt="image" src="https://user-images.githubusercontent.com/22511289/231641158-5bbbc226-3e56-4509-a914-49028107a19c.png">

Reply:
The tx should be `null` if `address` is invalid, or it will be hazardous as others believe the data and skip validation.

Reply:
It’s a potential risk due to legacy codebase and will be improved in future

Reply:
> The tx should be `null` if `address` is invalid, or it will be hazardous as others believe the data and skip validation.

Comments have been added.